src/main/scala/uk/gov/hmrc/ct/ct600/v3/formats/package.scala

```diff
@@ -24,16 +24,6 @@ package object formats {
 
   implicit val b1Format: Format[B1] = new StringFormat[B1](B1.apply)
 
-  implicit val b2Format: Format[B2] = new StringFormat[B2](B2.apply)
```

Comment:
B2 is now a linked field, removing formats for non-input fields is a good idea because it fails fast if you try to set or initialize the value (we have just a bug recently where an initializer was left in place for a field that had changed from input to calculated).